contracts/proxy/transparent/ProxyAdmin.sol

```diff
@@ -4,19 +4,19 @@
 pragma solidity ^0.8.20;
 
 import {ITransparentUpgradeableProxy} from "./TransparentUpgradeableProxy.sol";
-import {Ownable} from "../../access/Ownable.sol";
+import {Ownable2Step, Ownable} from "../../access/Ownable2Step.sol";
 
 /**
  * @dev This is an auxiliary contract meant to be assigned as the admin of a {TransparentUpgradeableProxy}. For an
  * explanation of why you would want to use this see the documentation for {TransparentUpgradeableProxy}.
  */
-contract ProxyAdmin is Ownable {
+contract ProxyAdmin is Ownable2Step {
```

Comment:
Same as https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4502/files#r1284069930

Reply:
Oh, this is an amazing reason!